src/passes/warnSupportedFeatures.ts

```diff
@@ -50,6 +75,15 @@ export class WarnSupportedFeatures extends ASTMapper {
       [...addresses.entries()].forEach(([path, nodes]) => warn(path, nodes));
     }
 
+    if (deploySalts.size > 0) {
+      console.log(
+        `${warning(
+          'Warning',
```

Comment:
@rodrigo-pino can we make a note of this in the docs as well

Reply:
There was already mention of it in `doc/features/contract_factories.mdx` so I updated that part